### PR TITLE
New friday schedule

### PR DESCRIPTION
**Pull request by reubengt**
Closes https://github.com/orgs/foundersandcoders/projects/7/views/8

### CODE REVIEW

File: src/_data/defaultSchedule.yml [hunk 1]
```diff
@@ -121,21 +117,21 @@ friday:
...
-    end: 15:15
+    end: 15:00
+    name: Presentation prep
+    url: /course/handbook/projects/#project-presentation
+  - start: 15:00
+    end: 15:30
+    name: Team SGC
+    url: /course/handbook/retrospectives/#team-retrospectives
+  - start: 15:30
+    end: 16:15
```
Comment (Albadylic) [inline src/_data/defaultSchedule.yml:128]: Maybe having SGC before presentations makes sense so they can talk about their workflow in their presentation?

File: src/_data/defaultSchedule.yml [hunk 2]
```diff
@@ -121,21 +117,21 @@ friday:
...
-  - start: 15:15
-    end: 16:00
-    name: Cohort SGC
-    url: /course/handbook/retrospectives/
-  - start: 16:00
-    end: 16:45
-    name: Team SGCs
-    url: /course/handbook/retrospectives/#team-retrospectives
-  - start: 16:45
+  - start: 16:15
```
Comment (Albadylic) [inline src/_data/defaultSchedule.yml:131]: Is 45 minutes enough for presentations?

In [our guidance](https://learn.foundersandcoders.com/course/handbook/project-presentations/), we recommend 12+5 minutes per presentation, so 51 minutes for three groups.

File: src/course/handbook/retrospectives.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ The whole cohort of students sit in a circle, if space allows, with their mentor
...
 
 ## Team retrospectives
 
 We also run a smaller "stop go continue" meeting with each project team so they are able to reflect on how their group project work went that week.
 
-During the first two weeks, FAC-staff will offer to facilitate team SGCs or at least ensure that each team has a facilitator and time keeper. The team may want to assign somebody as note taker and have the 4th person play a "reflective" role at the end of the session to provide a closing response to the SGC. The facilitator should not feel they need to respond to every issue and, except for reading out each point, doesn't talk more than the others
+The team may want to assign somebody as note taker and have the 4th person play a "reflective" role at the end of the session to provide a closing response to the SGC. The facilitator should not feel they need to respond to every issue and, except for reading out each point, doesn't talk more than the others
```
Comment (Albadylic) [inline src/course/handbook/retrospectives.md:29]:
```suggestion
The team may want to assign somebody as facilitator. The facilitator should not feel they need to respond to every issue and, except for reading out each point, doesn't talk more than the others
```

**Review by Albadylic — APPROVED** [review]

File: src/_data/defaultSchedule.yml (hunk 2 above)
Comment (reubengt) [reply src/_data/defaultSchedule.yml:131]: has been enough so far!! maybe we should change our guidance to 12+3

**Pull request merged**